jobtracker.go

```diff
@@ -77,8 +77,11 @@ func getJSON(url string, data interface{}) error {
 	if err != nil {
 		return err
 	}
-
-	return json.Unmarshal(jsonBytes, data)
+	err = json.Unmarshal(jsonBytes, data)
+	if err != nil {
+		log.Printf("Response is not valid JSON:\n%s", string(jsonBytes[:]))
```

Comment:
I think you want another `\n` at the end of the string? (Otherwise the next log is going to be on the same line.)

Reply:
Can we also print what the error is?

Reply:
i thought so too, but i was getting blank log lines and the logs look fine with this change.
it's possible i'm relying on `jsonBytes` ending with a newline though, should add a trailing newline to be double-sure?

Reply:
sgtm